en/frames_rover/differential_rover.md

```diff
@@ -32,82 +32,219 @@ To start using the differential-steering rover:
 
 3. Open the [Actuators Configuration & Testing](../config/actuators.md) to map the motor functions to flight controller outputs.
 
-This is sufficient to drive the the rover in [manual mode](../flight_modes_rover/index.md#manual-mode) (see [Flight modes](../flight_modes_rover/index.md)).
+This is sufficient to drive the the rover in [manual mode](../flight_modes_rover/manual.md#manual-mode) (see [Flight modes](../flight_modes_rover/index.md)).
 
 ::: info
-The parameter [RD_MAN_YAW_SCALE](../advanced_config/parameter_reference.md#RD_MAN_YAW_SCALE) can be used to scale the manual input for the yaw rate.
+The parameter [RD_MAN_YAW_SCALE](#RD_MAN_YAW_SCALE) can be used to scale the manual input for the yaw rate.
```

Comment:
Should this be moved below to the tuning?

Reply:
The entire tuning process is now structure into drive modes. I merged the "Basic Setup" into manual mode and also tried to elaborate on the _RD_MAN_YAW_SCALE_ parameter. See [here](https://github.com/PX4/PX4-user_guide/pull/3377/files#diff-2d1b0e12dead39732d61a37cb9ec99db8a3b3ed773308aeebae58d061c4703d9R11-R34).

Reply:
Cool. Though I split the basic setup again and made manual tuning minimal.